fhir-client/src/main/java/com/ibm/fhir/client/FHIRClient.java

```diff
@@ -344,7 +344,10 @@
      * @return a FHIRResponse that contains the results of the 'search-all' operation
```

Comment:
might want to add that this is specific to get

Reply:
go point, will add

Reply:
done